webrtc/videoroom/Dockerfile

```diff
@@ -0,0 +1,76 @@
+FROM elixir:1.11-alpine AS build
+ENV DEBIAN_FRONTEND=noninteractive
```

Comment:
Unnecessary if you use alpine

Reply:
Sure - a leftover from stages, where I've been using `elixir:1.11`, not yet an alpine image.